packages/flutter_image/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
+## 4.1.9
+
+* Adds an example app
```

Comment:
This needs a period, per the CHANGELOG style guide linked from the PR checklist.